src/lib/Toast/Toast.svelte

```diff
@@ -120,14 +127,16 @@
   .toast {
     padding: var(--toast-padding, 10px);
     font-size: var(--toast-font-size, 14px);
+    font-family: var(--toast-font-family);
+    font-weight: var(--toast-font-weight);
     height: var(--toast-height, auto);
     border-radius: var(--toast-border-radius, 0px);
     width: var(--toast-width, auto);
-    align-items: var(--toast-align-items, center);
```

Comment:
Why we have removed this default value?

Reply:
As we add a slot and subtext to the middle div, it will increase the height of the div and cause the icon to move to the center, which is not the expected outcome